libs/platform/src/lib/components/approval-flow/approval-flow-node/approval-flow-node.component.scss

```diff
@@ -187,6 +202,24 @@ $carousel-marker-top--edit-mode: $node-height--edit-mode / 2 - $carousel-marker-
             & + .#{$block}__drop-zone {
                 left: -1.8rem;
             }
+
+            &-all {
+                left: -3.5rem;
+
+                & + .#{$block}__drop-zone {
+                    left: -3.8rem;
+                }
+            }
+        }
+    }
+
+    &.#{$block}--last-blank {
+        .#{$block}__add--after-all {
+            left: calc(278px + 2.5rem);
```

Comment:
do we need the `calc` here since it can just be replaced with the value in rem?

Reply:
Refactored